jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/utils/Basics.java

```diff
@@ -0,0 +1,87 @@
+package com.epam.test_generator.api.steps.utils;
+
+import com.epam.http.response.RestResponse;
+import com.epam.test_generator.api.container.TestContext;
+import com.epam.test_generator.api.steps.StepBackground;
+import com.epam.test_generator.controllers.admin.request.UserRoleUpdateDTO;
+import com.epam.test_generator.controllers.project.response.ProjectDTO;
+import com.epam.test_generator.controllers.user.request.LoginUserDTO;
+import com.epam.test_generator.controllers.user.request.RegistrationUserDTO;
+import com.epam.test_generator.controllers.user.response.UserDTO;
+import com.epam.test_generator.dto.TokenDTO;
+import org.springframework.http.HttpStatus;
+
+import static com.epam.http.requests.RequestData.requestBody;
+import static com.epam.test_generator.api.ApiTokenInserter.requestBodyAndToken;
+import static com.epam.test_generator.api.ApiTokenInserter.requestDataAndToken;
+import static com.epam.test_generator.api.BddGeneratorApi.*;
+
+public class Basics extends StepBackground {
```

Comment:
Why you need to extend `StepBackground` here?

Reply:
For access to ObjectMapper